### DISCUSSION TITLE
Appointment rollback after cancel the edit Action

### DISCUSSION THREAD

**Discussion by jncalderon** [saade/filament-fullcalendar#208] [Q&A]
Hi guys, I have a question about the drag&drop, currently when I move the appointment show the modal to edit the record, but when I click on cancel/close button for the modal, basically cancel the edit, then the appointment doesn't rollback to the original place, how I can handle the cancel operation?

**Comment by jncalderon** [comment]
@saade hi any idea? to be honest I don't have an idea just to generate a PR with the improvement, but even a workaround could be catch the event when a modal is canceled or closed to dispatch:
```php
$this->dispatch('filament-fullcalendar--refresh');
```

**Comment by LeamonaDor1** [comment]
[Like this](https://github.com/saade/filament-fullcalendar/discussions/218#discussioncomment-10527222)
